hardware/edid.ksy

```diff
@@ -62,6 +62,16 @@ seq:
       "established timings".
     repeat: expr
     repeat-expr: 8
+  - id: eighteen_byte_descriptors
+    size: 18
+    type: dtd_timing
+    doc: |
+      Array of 18 byte descriptors, which the first of them shall be
+      Detailed Timing Descriptor reflecting Preferred Timing Mode and
+      each of the rest is either a Detailed Timing Descriptor or
+      a Display Descriptor.
+    repeat: expr
+    repeat-expr: 4
```

Comment:
`doc` should come after everything (see https://doc.kaitai.io/ksy_style_guide.html#seq-attr):

```suggestion
    repeat: expr
    repeat-expr: 4
    doc: |
      Array of 18 byte descriptors, which the first of them shall be
      Detailed Timing Descriptor reflecting Preferred Timing Mode and
      each of the rest is either a Detailed Timing Descriptor or
      a Display Descriptor.
```

Reply:
Fixed in https://github.com/N-Nagorny/kaitai_struct_formats/commit/89ae655489f2d4a500ab8a099a6b50aeced93edc